test/integration/patients/sidebar/action-sidebar.js

```diff
@@ -1653,11 +1653,6 @@ context('action sidebar', function() {
         fx.data.relationships.role = { data: { id: '66666' } };
         return fx;
       })
-      .routeSettings(fx => {
-        fx.data.push({ id: 'upload_attachments', attributes: { value: true } });
-
-        return fx;
-      })
```

Comment:
Removing this causes a coverage drop here: https://coveralls.io/builds/62831838/source?filename=src%2Fjs%2Fentities-service%2Fentities%2Factions.js#L184.

We added this in a recent PR: https://github.com/RoundingWell/care-ops-frontend/pull/1135